travel-provider/TSP-30-RankTravelOptions/TSP-30-RankTravelOptions-RequestSchema.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "TSP-30-RankTravelOptionsRequest Request Schema",
+  "version": "0.0.1",
+  "description": "Schema for ranking travel options request sent by CWT",
+  "properties": {
+    "returnId": {
+      "type": "integer",
+      "description": "Unique Id for this booking - generated by RMP."
+    },
+    "travelOptions": {
+      "type": "array",
+      "description": "A list of 1 to 9 travel options.",
+      "minItems": 1,
+      "maxItems": 9,
+      "items": {
+        "$ref": "#/definitions/travelOption"
+      }
+    },
+    "providerFeedback": {
+      "type": "object",
+      "description": "If fewer than 9 travel options are provided the reason must be specified.",
+      "properties": {
+        "limitedOptionsReason": {
+          "description": "The reason why less than 9 travel options have been provided",
+          "type": "string",
+          "enum": ["PERMITTED_AIRLINE_CONSTRAINTS", "FLIGHT_QUOTA_CONSTRAINTS", "FLIGHT_FREQUENCY_CONSTRAINTS", "REQUEST_REQUIREMENT_CONSTAINTS", "ESCORT_PROVIDER_CONSTRAINTS", "OTHER"]
+        },
+        "limitedOptionsDescription": {
+          "description": "A detailed description of why there are less than 9 travel options",
+          "type": "string"
+        }
+      },
+      "required": [
+        "limitedOptionsReason"
+      ]
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "returnId",
+    "travelOptions"
+  ],
+  "definitions": {
+    "travelOption": {
+      "type": "object",
+      "description": "A travel option that meets the requirements of the return case.",
+      "properties": {
+        "flights": {
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/flight"
+          }
+        },
+        "totalTicketPricePence": {
+          "type": "integer"
+        }
+      },
+      "required": [
+        "flights",
+        "totalTicketPricePence"
+      ]
+    },
+    "flight": {
+      "type": "object",
+      "description": "Flight details.",
+      "properties": {
+        "flightNumber": {
+          "type": "string"
+        },
+        "departureDateTime": {
+          "type": "string",
+          "format": "date-time"
+        }
+      },
+      "required": [
+        "flightNumber"
+      ]
```

Comment:
Add  `"additionalProperties": false` for this object (it only applies to the immediate object you specify it in, not child objects).